web/Views/Property/PropertyInputForm/TenantPreferences.cshtml

```diff
@@ -7,15 +7,15 @@
 
     <div class="row g-3">
         <div class="col-auto d-grid">
-            @Html.CheckBox("AcceptsSingleTenant", Model.AcceptsSingleTenant ?? false, new { @class = "btn-check" })
-            <label asp-for="AcceptsSingleTenant" class="btn btn-outline-blue shadow-none pill-check">Single tenant</label>
+            <input type="checkbox" asp-for="AcceptsSingleTenant" class="btn-check"/>

```

Comment:
Happy to use tags, we can also use Html.CheckBoxFor if you prefer the other method (I am very happy either way :) )